src/zjs_net.c

```diff
@@ -864,22 +862,22 @@ static ZJS_DECL_FUNC(net_create_server)
     zjs_obj_add_boolean(server, false, "listening");
     zjs_obj_add_number(server, NET_DEFAULT_MAX_CONNECTIONS, "maxConnections");
 
-    server_handle_t *handle = zjs_malloc(sizeof(server_handle_t));
-    if (!handle) {
+    server_handle_t *server_h = zjs_malloc(sizeof(server_handle_t));
+    if (!server_h) {
         jerry_release_value(server);
         return zjs_error("could not alloc server handle");
     }
 
-    handle->server = server;
-    handle->listening = 0;
```

Comment:
Is this handle->listening = 0; line no longer needed?

Reply:
No because I'm `memset`ting the whole struct to 0 on the next line now.